exchange/rpc.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"github.com/golang/glog"
 	"github.com/open-horizon/anax/config"
+	"github.com/open-horizon/anax/persistence"
```

Comment:
Yes, I agree with you. I have tried different ways to avoid doing this. Let's talk about it.